airbyte-integrations/connectors/source-marketo/source_marketo/source.py

```diff
@@ -447,6 +447,12 @@ def request_params(
     def parse_response(self, response: requests.Response, stream_state: Mapping[str, Any], **kwargs) -> Iterable[Mapping]:
         for record in super().parse_response(response, stream_state, **kwargs):
             record["updatedAt"] = record["updatedAt"][:-5]  # delete +00:00 part from the end of updatedAt
+
+            # Remove non parser-friendly suffix from createdAt
+            created_at = record.get("createdAt", "")
+            if created_at[-5:] == "+0000":
```

Comment:
Curious why this logic is subtly different from similar logic at line 449 for updatedAt?

Reply:
It fundamentally does the same as the `updatedAt` logic, except I don't think `createdAt` exists in all of the Marketo data sources. Also, I want to make sure that we only replace the suffix if it's problematic and not just slice off the last 5 characters. 

tl;dr: Erring on the "better safe than sorry" side for this one

Reply:
yeah but why is the format of the timestamp youre chopping off different?

Reply:
In a few instances, I didn't see the `+0000` in the source value, so those cases might chop off too much.